client/src/components/RepoList/RepoListItem/RepoListItem.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import { RepoListItemWrapper, RepoListItemItems } from './RepoListItem.styles';
+
+const RepoListItem = ({ username, repositories_count }) => {
+    return(
+        <RepoListItemWrapper>
+            <RepoListItemItems>Github nick: {username}</RepoListItemItems>
+            <RepoListItemItems>Number of repositories: {repositories_count}</RepoListItemItems>
```

Comment:
Why not add parameters like: language, number of stars, name of repositories?

Reply:
I would be better for sure, but if I correctly understood the backend/app/models.py file this is what we currently get from the endpoint